st2actions/conf/logging.conf

```diff
@@ -21,13 +21,13 @@ args=(sys.stdout,)
 class=st2common.log.FormatNamedFileHandler
 level=INFO
 formatter=verboseConsoleFormatter
-args=('logs/st2actionrunner.{pid}.log',)
+args=('logs/st2actionrunner.{pack}.log',)
```

Comment:
ActionRunner itself logs when not in context of running an action in a pack. Per pack logging make very little sense in that case.